app/components-react/shared/PlatformLogo.tsx

```diff
@@ -44,7 +44,7 @@ function getTikTokSvg(color: string) {
 
 function TikTokLogo(p: IProps) {
   const color = 'black';
-  const size = 40;
+  const size = (p.size && (sizeMap[p.size] || p.size)) || 40;
```

Comment:
When working with numbers doing null/undefined fallbacks it's better to use `??` instead of `||`.

Reply:
was just copying over the existing code, but can make that change. in this case though i don't think size is ever expected to be 0

Reply:
Sure makes sense.  Just mentioned because general good practice.